.eslintrc

```diff
@@ -7,7 +7,6 @@
     "no-underscore-dangle": 0,
     "no-use-before-define": [2, "nofunc"],
     "max-len": [2, 150, 2],
-    "quotes": [2, "single"],
```

Comment:
My prettier modifie all files with `" "` instead of `' '` and
I can't do this : 

``` 
"quotes": [2, "single", "double"]
```
So I removed it instead of revert all prettier modifications